### ISSUE TITLE
RFE: default product if unspecified

### ISSUE THREAD

**Issue by ktdreyer** [ktdreyer/helga-productpages#4]
```
< ktdreyer>: kraken: 3.0z1 release date
< kraken>: ktdreyer, I could not find release 3.0z4-release in pp.engineering.redhat.com
< ktdreyer>: kraken: rhcs 3.0z1 release date
< kraken>: ktdreyer, Red Hat Ceph Storage 3.0 z4 is on (date)
```

It would be faster to default to the "ceph" product if one is unspecified.

**Comment by ktdreyer** [comment]
Fixed in 55c803fa0946ea8e40050fbfccdb46667804236f

**Comment by ktdreyer** [comment]
tagged v1.2.0
